packages/nitro-protocol/src/index.ts

```diff
@@ -14,6 +16,16 @@ export const ContractArtifacts = {
   AssetHolderArtifact, // TODO do we want to export this?
 };
 
+/**
+ * Various test contract artifacts used for testing
+ * They expose helper functions to allow for easier testing
+ * They should NEVER be used in a production environment.
+ */
+export const TestContractArtifacts = {
+  TestNitroAdjudicatorArtifact,
+  TestAssetHolderArtifact,
```

Comment:
Is this used anywhere?

Reply:
`TestNitroAdjudicatorArtifact` is used in the tests but may not be needed if we can just set `evm_mine` timestamp